aioftp/server.py

```diff
@@ -1318,6 +1319,20 @@ def get_paths(self, connection, path):
         connection.response(code, info)
         return True
 
+    @ConnectionConditions(ConnectionConditions.login_required)
+    async def pbsz(self, connection, rest):
+        connection.response('200', '')
+        return True
+
+    @ConnectionConditions(ConnectionConditions.login_required)
+    async def prot(self, connection, rest):
+        if rest == 'P':
+            code, info = '200', ''
+        else:
+            code, info = '502', ''
+        connection.response(code, info)
+        return True
+
```

Comment:
What for this methods are? According project codestyle please replace single quotes with double one.

Reply:
These are for compatibility with ftps connection esablishing.
Prot P is determinate that data connection works under ssl.

Reply:
Pbsz is for protection buffer size which is usually 0

Reply:
As mentioned above, you should change codestyle.

Reply:
Sorry...I dont know how to modify this pull request.....But its too late to do, good night.

Reply:
You do not need to modify something. You should add commit to your branch and it will appear here.

Reply:
> You do not need to modify something. You should add commit to your branch and it will appear here.

Ok, I have commited.